release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -3725,3 +3725,33 @@ $ oc adm release info 4.13.31 --pullspecs
 [id="ocp-4-13-31-updating"]
 ==== Updating
 To update an existing {product-title} 4.13 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI].
+
+[id="ocp-4-13-32"]
+=== RHSA-2024:0660 - {product-title} 4.13.32 bug fix and security update
+
+Issued: 2024-02-07
+
+{product-title} release 4.13.32, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2024:0660[RHSA-2024:0660] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2024:0662[RHBA-2024:0662] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.13.32 --pullspecs
+----
+
+[id="ocp-4-14-11-features"]
+==== Features
+
+The following feature is included in this z-stream release:
+
+[id="ocp-4-13-32-wherabouts-cron-schedule"]
+===== Enabling configuration of whereabouts cron schedule
+
+* The Whereabouts reconciliation schedule was hard-coded to run once per day and could not be reconfigured. With this release, a ConfigMap has enabled the configuration of the whereabouts cron schedule. xref:../networking/multiple_networks/configuring-additional-network.adoc#nw-multus-creating-whereabouts-reconciler-daemon-set_configuring-additional-network[Creating a Whereabouts reconciler daemon set].
```

Comment:
- "Hardcoded" one word per IBM SG
- Markup `ConfigMap` and add "object" or "resource" after
- Initcap "Whereabouts"
- Feels like the xref is missing a "For more information" or something to make it a complete sentence?

```suggestion
* The Whereabouts reconciliation schedule was hardcoded to run once per day and could not be reconfigured. With this release, a `ConfigMap` object has enabled the configuration of the Whereabouts cron schedule. For more information, see xref:../networking/multiple_networks/configuring-additional-network.adoc#nw-multus-creating-whereabouts-reconciler-daemon-set_configuring-additional-network[Creating a Whereabouts reconciler daemon set].
```